2Weekly Mission/Linkbrarystyle.css

```
@@ -0,0 +1,387 @@
+}
+
+.cta {
+  display: flex;
+  width: 128px;
+  padding: 16px 20px;
+  justify-content: center;
+  align-items: center;
+  border-radius: 8px;
+  background: linear-gradient(91deg, #6d6afe 0.12%, #6ae3fe 101.84%);
+  color: #f5f5f5;
+  font-family: Pretendard;
+  font-size: 18px;
+  font-style: normal;
+  font-weight: 600;
+  line-height: normal;
+}
+
+.content1 {
+  display: flex;
+  margin-top: 93px;
+  margin-bottom: 70px;
+  padding: 70px 16.7% 0px;
+  flex-direction: column;
+  justify-content: flex-end;
+  align-items: center;
+  background-color: var(--gray5);
+  text-wrap: nowrap;
+  overflow: hidden;
+}
+
+.container1 {
+  display: flex;
+  flex-direction: column;
+  gap: 40px;
+  align-items: center;
+  text-align: center;
+  max-height: 833px;
+  max-width: 1200px;
+}
+
+.container1 > h2 {
+  color: #000;
+  font-family: Pretendard;
+  font-size: 64px;
+  font-style: normal;
+  font-weight: 900;
+  line-height: 125%;
+}
+
+.container1 > h2 > span {
+  color: transparent;
+  background: linear-gradient(91deg, #6d6afe 17.28%, #ff9f9f 74.98%);
+  -webkit-background-clip: text;
+  -webkit-text-fill-color: transparent;
+}
+
+.link {
+  width: 350px;
+}
+
+.image-box1 {
+  padding: 50px 41px 0;
+}
+
+.image-box1 > img {
+  width: 100%;
+}
+
+.content2 {
+  padding: 50px 24%;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+.frame {
+  display: flex;
+  justify-content: flex-start;
+  align-items: center;
+  gap: 157px;
+}
+
+.ad-text {
+  color: #6b6b6b;
+  font-family: Abel;
+  font-size: 16px;
+  font-style: normal;
+  font-weight: 400;
+  line-height: 150%; /* 24px */
+  width: 291px;
+  display: flex;
+  flex-direction: column;
+  gap: 10px;
+}
+
+.ad-text h3 {
+  color: #000;
+  font-family: Pretendard;
+  font-size: 48px;
+  font-style: normal;
+  font-weight: 700;
+  line-height: normal;
+  letter-spacing: -0.3px;
+}
+
+.adText-deco1 {
+  background: linear-gradient(96deg, #fe8a8a 1.72%, #a4ceff 74.97%);
+  background-clip: text;
+  -webkit-background-clip: text;
+  -webkit-text-fill-color: transparent;
+}
+
+.adText-deco2 {
+  background: linear-gradient(277deg, #6fbaff 59.22%, #ffd88b 93.66%);
+  background-clip: text;
+  -webkit-background-clip: text;
+  -webkit-text-fill-color: transparent;
+}
+
+.adText-deco3 {
+  background: linear-gradient(
+    99deg,
+    #6d7ccd 19.76%,
+    rgba(82, 136, 133, 0.22) 52.69%
+  );
+  background-clip: text;
+  -webkit-background-clip: text;
+  -webkit-text-fill-color: transparent;
+}
+
+.adText-deco4 {
+  background: linear-gradient(271deg, #fe578f -9.84%, #68e8f9 107.18%);
+  background-clip: text;
+  -webkit-background-clip: text;
+  -webkit-text-fill-color: transparent;
+}
+
+.ad-image {
+  width: 550px;
+  height: 450px;
+  background-color: var(--gray5);
+  border-radius: 15px;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  overflow: hidden;
+}
+
+.image1 img {
+  width: 560px;
+}
+
+.image2 {
+  padding-top: 30px;
+}
+.image2 img {
```

규칙에 따라 공백도 정확하게 지켜주세요! 

```suggestion
}

.image2 img {
```